packages/visual-sdk/src/store/EditorStore.spec.ts

```diff
@@ -22,6 +22,21 @@ describe('EntityStore', () => {
     expect(createStore().entities).toEqual(entities);
   });
 
+  it('prevents id collision for assets and entries', async () => {
+    const patchedAsset = { ...asset, sys: { ...asset.sys, id: '1' } };
+    const patchedEntry = { ...entry, sys: { ...entry.sys, id: '1' } };
+
+    const store = new EntityStore({ entities: [patchedAsset, patchedEntry], locale });
+
+    const [resolvedAsset, resolvedEntry] = await Promise.all([
+      store.fetchAsset('1'),
```

Comment:
What happens with?

```
public get entities() {
  return [...this.entryMap.values(), ...this.assetMap.values()];
}
```

Reply:
() is it possible to have duplicates? EntityStore

Reply:
yes there it is possible to have duplicate `sys.id`, but you also have the `sys.type` to differentiate between them, so that should be fine :)